src/components/modal/modal.njk

```diff
@@ -0,0 +1,84 @@
+            </li>
+          </ul> 
+        </div>
+      </div>
+      <button class="usa-button usa-modal__close" aria-label="Close this window"  data-close-modal>
+        <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
+          <use xlink:href="{{ uswds.path }}/img/sprite.svg#close"></use>
+        </svg>
+      </button>
+    </div>
+  </div>
+</div>
+
+
+<div class="margin-y-3">
+  <a href="#modal-2" class="usa-button" aria-controls="modal-2" data-open-modal>Large modal</a>
+  <div class="usa-modal usa-modal--lg" id="modal-2" aria-labelledby="modal-2-heading" aria-describedby="modal-2-description">
+    <div class="usa-modal__content">
+      <div class="usa-modal__main">
+        <h2 class="usa-modal__heading" id="modal-2-heading">Are you sure you want to continue?</h2>
+        <div class="usa-prose">
+          <p id="modal-2-description">You have unsaved changes that will be lost.</p>
+        </div>
+        <div class="usa-modal__footer">
+          <ul class="usa-button-group">
+            <li class="usa-button-group__item">
+              <button type="button" class="usa-button" data-close-modal>Continue without saving</button>
+            </li>
+            <li class="usa-button-group__item">
+              <button type="button" class="usa-button usa-button--unstyled padding-105 text-center" data-close-modal>Go back</button>
+            </li>
+          </ul> 
+        </div>
+      </div>
+      <button class="usa-button usa-modal__close" aria-label="Close this window"  data-close-modal>
+        <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
+          <use xlink:href="{{ uswds.path }}/img/sprite.svg#close"></use>
+        </svg>
+      </button>
+    </div>
+  </div>
+</div>
+
+<div class="margin-y-3">
+  <a href="#modal-3" class="usa-button" aria-controls="modal-3" data-open-modal>Modal with forced action</a>
+  <div class="usa-modal" id="modal-3" aria-labelledby="modal-3-title" aria-describedby="modal-3-description" data-force-action>
+    <div class="usa-modal__content">
+      <div class="usa-modal__main">
+        <h2 class="usa-modal__heading" id="modal-3-title">Your session will end soon.</h2>
+        <div class="usa-prose">
+          <p id="modal-3-description">You’ve been inactive for too long. Please choose to stay signed in or sign out. Otherwise, you’ll be signed out automatically in 5 minutes.</p>
+        </div>
+
+        <div class="usa-modal__footer">
+        <ul class="usa-button-group">
+          <li class="usa-button-group__item">
+            <button type="button" class="usa-button" data-close-modal>Yes, stay signed in</button>
+          </li>
+          <li class="usa-button-group__item">
+            <button type="button" class="usa-button usa-button--unstyled padding-105 text-center" data-close-modal>Sign out</button>
+          </li>
+        </ul> 
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
We should create settings for this component so we can use it like a real component. See [Search](https://github.com/uswds/uswds/blob/accelerator-3936/modal/src/components/search/search.njk) for example.

Reply:
Added in c6bcc42.